source/Halibut/ServiceEndPoint.cs

```diff
@@ -8,11 +8,15 @@ public class ServiceEndPoint : IEquatable<ServiceEndPoint>
     {
         readonly string baseUriString;
 
+        // TODO - ASYNC ME UP!
```

Comment:
In another PR we will need a ServiceEndPoint constructed with the halibuttimeoutsandlimits. Or perhaps the halibut runtime should have a ServiceEndPoint factory with timeouts configured from its limits